### PR TITLE
Add CSV Databases pages

### PR DESCRIPTION
**Pull request by kevcenteno**

### CODE REVIEW

**Comment by kevcenteno** [comment]
@wesrice minFraud and geoip web services pages have broken link buttons to the request and response pages.  The subnav menu item is fine though

**Comment by kevcenteno** [comment]
@wesrice Nevermind, I'll fix it.

File: src/components/CsvBlockTable.tsx [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+          )}
+        </Tr>
+        <Tr>
+          <Td>postal_code</Td>
+          <Td>string</Td>
+          <Td>
+            A postal code close to the user&apos;s location. We return the
+            first 3 characters for Canadian postal codes. We return the first
+            2-4 characters (outward code) for postal codes in the United
+            Kingdom.
```
Comment (rajnihatti) [inline src/components/CsvBlockTable.tsx:203]: On the current site we have the following info for the number of char returned for each country. Does that need to be added here?
```A postal code close to the user's location. For the following countries, we return partial postal codes with the number of characters indicated below:

United States: 5
Canada: 3
United Kingdom: 2-4
Brazil: 5
Ireland: 3
Japan: 7 (specified for the first 6. The last digit defaults to 1)
Netherlands: 4
Portugal: 7 (accurate for the first 4. The last 3 often defaults to -001)
Singapore: 2

File: src/components/CsvBlockTable.tsx [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>accuracy_radius</Td>
+          <Td>integer</Td>
+          <Td>
+            The radius in kilometers around the specified location where the IP
```
Comment (rajnihatti) [inline src/components/CsvBlockTable.tsx:277]: The original text says: 
The approximate accuracy radius, in kilometers, around the latitude and longitude for the geographical entity (country, subdivision, city or postal code) associated with the IP address. We have a 67% confidence that the location of the end-user falls within the area defined by the accuracy radius and the latitude and longitude coordinates.

Not sure if this was changed on purpose. If so you can ignore this

**Comment by wesrice** [comment]
@rajnihatti  I updated those descriptions and pushed.

File: src/components/CsvBlockTable.tsx (hunk 1 above)
Comment (rajnihatti) [reply src/components/CsvBlockTable.tsx]: I think I see where the two different descriptions came from. On the https://dev.maxmind.com/geoip/geoip2/geoip2-city-country-csv-databases/ page it shows the text I added above for `postal_code`, but on the https://dev.maxmind.com/geoip/geoip2/geoip2-enterprise-csv-database/ page it shows the text you had originally. I am not sure if they need to remain different?

Comment (rajnihatti) [reply src/components/CsvBlockTable.tsx]: Also just FYI the font needs to match up
image

File: src/components/CsvBlockTable.tsx (hunk 2 above)
Comment (rajnihatti) [reply src/components/CsvBlockTable.tsx]: same issue as the comment above. On the https://dev.maxmind.com/geoip/geoip2/geoip2-city-country-csv-databases/ page it shows the text I added above for `accuracy_radius`, but on the https://dev.maxmind.com/geoip/geoip2/geoip2-enterprise-csv-database/ page it shows the text you had originally.

Comment (rajnihatti) [reply src/components/CsvBlockTable.tsx]: maybe they are supposed to be different?

Comment (christophermluna) [reply src/components/CsvBlockTable.tsx]: i'll make a note of this and we'll standardize it during UAT or after the site goes live

**Pull request merged**